root/types.js

```diff
@@ -57,6 +57,15 @@ declare type OptionTreeT = {|
   +description: string,
 |};
 
+declare type WorkT = EntityT & {
+    l_type_name?: string;
+};
+
+declare type URLT = EntityT & {
+    decoded: string;
+    edits_pending: bool;
```

Comment:
In another branch I created an `EditsPendingT` to shared between multiple types. Seems like a good idea to add it now to avoid conflicts later:
```JavaScript
declare type EditsPendingT = {|
  +editsPending: boolean,
|};

[...]

declare type URLT =
  & EditsPendingT
  & EntityT;
```